ChurchCRM/CRM
# attempt to fix potentially-undefined fn apache_get_modules, only allow upgrade when logged in, fix kiosk routes, misc minor cleanup

# Description
Please include a summary of the changes and the related issue. Please also include relevant motivation and context.

* Hopefully addresses #6621
* added use statement for `\Slim\App` so PHP can find the class for the kiosk routes (#6628)
* also noticed anyone can currently upgrade the system without being logged in, so I restricted that.

## Issue ticket number and link

If anyone is experiencing #6621, feel free to test this.
Closes #6628

#### Screenshots (if appropriate)

N/A

#### How should this be manually tested?

* Unclear how to reproduce #6621
* Hit a kiosk route and see it work (resolves #6628)
* you can test the upgrade by going to the page unauthorized and then again when logged in.

## Type of change


# Checklist:
